simulation.py

```diff
@@ -0,0 +1,92 @@
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+PRICE_DATASET = 'price_dataset'
+COST_FUNCTION = 'cost_function'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays_all = {
+        region : raw_data['workdays'][region]
+        for region in TRADING_REGIONS if region in raw_data['workdays']}
+    bound_data = {}
+    for region, workdays in workdays_all.items():
+        bound_data[region] = copy.deepcopy(raw_data)
+        for dataset in bound_data[region].values():
+            dataset.set_workdays(workdays)
+    bound_data['workdays'] = workdays_all
+    return bound_data
+
+def extract_data(raw_data, data_list, price_dataset, region):
+    """
+    Extract data on data_list from raw_data.
+
+    Return data will be a dictionary, including three key-value pairs:
+    (1) 'workdays'
+        a DataController, the workdays data of the trading region.
+    (2) 'price'
+        a Dataset, the price dataset of the trading region.
+    (3) 'others'
+        a dict of str Dataset, other data that users specify in data_list.
+    """
+    return {
+        'workdays' : raw_data['workdays'][region],
+        'target_data' : raw_data[region][price_dataset],
+        'others' : {
+            name : raw_data[region][name]
+            for name in data_list if name in raw_data[region]}}
+
+def simulate(alpha_settings_path, ignore_evaluate_result):
+    """
+    Act the process of simulating.
+    """
+    try:
+        settings = importlib.import_module(alpha_settings_path).AlphaSettings()
+    except FileNotFoundError as error:
+        raise FileNotFoundError("Alpha does not exist.") from error
+    except ImportError as error:
+        raise ImportError("No available AlphaSettings in %s"
+                          % alpha_settings_path) from error
+    data_all = initialize()
+    _, region = settings.target
+    data_required = extract_data(
+        data_all, settings.data_list,
+        TRADING_CONFIGS[settings.target][PRICE_DATASET], region)
+    cost_function = TRADING_CONFIGS[settings.target][COST_FUNCTION](
+        region, TRADING_INSTRUMENTS[settings.target])
+    # results would put into evaluator after import evaluator.
+    results = Simulator(settings, data_required, cost_function).run()
+    if ignore_evaluate_result:
+        #This judgement will not dump evaluate result, after import evaluator.
+        pass
+    return results
+
+def main():
+    """
+    Run the simulation process.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'alpha_settings_path', help='Please import a path of your alpha.')
+    parser.add_argument(
+        '--ignore', '--ignore_evaluate_result',
+        help='Input this key words while not caring the evaluate result',
+        action='store_true')
+    args = parser.parse_args()
```

Comment:
Is it possible to add one more argument, `-o --output`, which specifies the output file path?
If this argument is set, `simulation_results` will be dumped to the file (with pickle); otherwise, the default behavior is to print it to stdout, which is what you do now.